ui/client/components/graph/node-modal/editors/expression/TextareaEditor.tsx

```diff
@@ -0,0 +1,43 @@
+import {Formatter, FormatterType, typeFormatters} from "./Formatter"
+import {Editor} from "./Editor"
+import i18next from "i18next"
+import React from "react"
+import Textarea from "../field/Textarea"
+
+type Props = {
+    expressionObj: $TodoType,
+    onValueChange: Function,
+    className: string,
+    formatter: Formatter,
+}
+
+const TextareaEditor: Editor<Props> = (props: Props) => {
```

Comment:
can we extract common stuff with text input? switchableTo, (non)switchableToHint are the same?